[2024-09-14T18:27:10.459+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: save_city_info_weather_v01.insert_data manual__2024-09-14T18:27:08.114068+00:00 [queued]>
[2024-09-14T18:27:10.466+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: save_city_info_weather_v01.insert_data manual__2024-09-14T18:27:08.114068+00:00 [queued]>
[2024-09-14T18:27:10.466+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-09-14T18:27:10.466+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2024-09-14T18:27:10.466+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-09-14T18:27:10.476+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_data> on 2024-09-14 18:27:08.114068+00:00
[2024-09-14T18:27:10.481+0000] {standard_task_runner.py:55} INFO - Started process 4391 to run task
[2024-09-14T18:27:10.488+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'save_city_info_weather_v01', 'insert_data', 'manual__2024-09-14T18:27:08.114068+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/saver_info.py', '--cfg-path', '/tmp/tmps9i1jnmd']
[2024-09-14T18:27:10.489+0000] {standard_task_runner.py:83} INFO - Job 67: Subtask insert_data
[2024-09-14T18:27:10.533+0000] {task_command.py:389} INFO - Running <TaskInstance: save_city_info_weather_v01.insert_data manual__2024-09-14T18:27:08.114068+00:00 [running]> on host 0c5fe11d0c9c
[2024-09-14T18:27:10.570+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=save_city_info_weather_v01
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2024-09-14T18:27:08.114068+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-09-14T18:27:08.114068+00:00
[2024-09-14T18:27:10.570+0000] {sql.py:253} INFO - Executing: 
            insert into weathers (city_name, max_temp, min_temp, wind, description) values (0,'MSK', '10', '6', '2', 'Kakto')
        
[2024-09-14T18:27:10.575+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-14T18:27:10.678+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-14T18:27:10.683+0000] {sql.py:364} INFO - Running statement: 
            insert into weathers (city_name, max_temp, min_temp, wind, description) values (0,'MSK', '10', '6', '2', 'Kakto')
        , parameters: None
[2024-09-14T18:27:10.685+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 261, in execute
    return_last=self.return_last,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 338, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 369, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: INSERT has more expressions than target columns
LINE 2: ...wind, description) values (0,'MSK', '10', '6', '2', 'Kakto')
                                                               ^

[2024-09-14T18:27:10.691+0000] {taskinstance.py:1327} INFO - Marking task as UP_FOR_RETRY. dag_id=save_city_info_weather_v01, task_id=insert_data, execution_date=20240914T182708, start_date=20240914T182710, end_date=20240914T182710
[2024-09-14T18:27:10.697+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 67 for task insert_data (INSERT has more expressions than target columns
LINE 2: ...wind, description) values (0,'MSK', '10', '6', '2', 'Kakto')
                                                               ^
; 4391)
[2024-09-14T18:27:10.709+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2024-09-14T18:27:10.719+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
